knowledge_graphs/parse_repo_into_neo4j.py

```
@@ -0,0 +1,280 @@
+    async def initialize(self):
+        """Initialize the Neo4j connection."""
+        try:
+            self.driver = AsyncGraphDatabase.driver(
+                self.neo4j_uri, 
+                auth=(self.neo4j_user, self.neo4j_password)
+            )
+            # Test the connection
+            async with self.driver.session() as session:
+                await session.run("RETURN 1")
+            logger.info("Neo4j connection initialized successfully")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)